octobot_trading/exchanges/types/rest_exchange.py

```diff
@@ -637,6 +637,13 @@ async def get_symbol_leverage(self, symbol: str, **kwargs: dict):
         """
         raise NotImplementedError("get_symbol_leverage is not implemented")
 
+    async def get_leverage_tiers(self, symbols: str = None, **kwargs: dict):
```

Comment:
Can you add the method declaration in AbstractExchange please?

Reply:
done